Source/GSXibLoading.m

```diff
@@ -158,3 +194,831 @@ - (id) initWithCoder: (NSCoder *)coder
   return self;
 }
 @end
+
+@implementation FirstResponder
```

Comment:
Class name without prefix? Is this correct? It seems like it was in the old code, too, so it's fine, but I'm just wondering.

Reply:
This gets explained in the corresponding header, which wasn't changed in the commit so you do not see it.
At the time when I wrote this code I was still trying to be clever, which I hopefully stopped doing :-) There is an object in XIB files that has the name FirstResponder. With this class in place that object gets correctly decoded as nil. Sorry, I promise never to do it again.